cors/src/main/java/io/helidon/cors/CorsSupportHelper.java

```diff
@@ -239,8 +238,10 @@ public CorsSupportHelper<Q, R> build() {
 
             CorsSupportHelper<Q, R>  result = new CorsSupportHelper<>(this);
 
-            LOGGER.log(System.Logger.Level.INFO,
-                    () -> String.format("CorsSupportHelper configured as: %s", result.toString()));
+            if (LOGGER.isLoggable(System.Logger.Level.DEBUG)) {
```

Comment:
I though that using a lambda in LOGGER.log avoids to use the `if (LOGGER.isLoggable(System.Logger.Level.DEBUG)) {`. Why is this needed?.